### ISSUE TITLE
Default Aspect Ratios — Last one is missing

### ISSUE THREAD

**Issue by davidhellmann** [nystudio107/craft3-imageoptimize#31]
```
'defaultAspectRatios' => [
    ['x' => 16, 'y' => 10],
    ['x' => 16, 'y' => 9],
    ['x' => 4, 'y' => 3],
    ['x' => 1, 'y' => 1],
    ['x' => 10, 'y' => 16],
    ['x' => 3, 'y' => 4],
    ['x' => 3, 'y' => 4],
],< 
```

![image](https://user-images.githubusercontent.com/764292/34664204-00f1c668-f45b-11e7-9df3-0a1a0c7cff67.png)

When I double the last line it works as expected.

**Comment by nielsnuebel** [comment]
https://github.com/nystudio107/craft3-imageoptimize/blob/master/src/fields/OptimizedImages.php#L237

https://github.com/nystudio107/craft3-imageoptimize/blob/master/src/templates/_components/fields/OptimizedImages_settings.twig#L99
this is the problem

**Comment by nielsnuebel** [comment]
https://github.com/nystudio107/craft3-imageoptimize/pull/32 Here is an idea to fix

**Comment by khalwat** [comment]
So this was actually a flaw in the `OptimizedImages_settings.twig` template, the logic in the field was correct @nielsnuebel . The problem was this:

```
                                        {% if aspectRatio.break is defined and aspectRatio.break %}
                                            </div><div>
                                        {% else %}
```

In other words, if the aspect ratio had the `break` attribute in the array, it would skip rendering the aspect ratio entirely.

I've pushed a fix; it'll be evident in version 1.4.8 when it is released (hopefully soon, I'm just tracking down a few other issues).
